### DISCUSSION TITLE
Error compiling "Your Configuration_adv.h file is for a newer version of Marlin

### DISCUSSION THREAD

**Discussion by MaynardMann** [mriscoc/Ender3V2S1#365] [Q&A] [answered]
Trying to compile the MM version for my s1 with f4 chip in VS Code and it keeps kicking back this error no idea what to do I'm using the project folder from the release page here https://github.com/mriscoc/Ender3V2S1 should I be using a different release for this config
![image](https://user-images.githubusercontent.com/60556984/189601501-1800661c-d226-4ab2-b063-c9bf1db5917f.png)

**Comment by mriscoc** [comment, marked as answer]
Yes, sorry. I'm in the process of releasing a new version with the current Marlin bugfixes and I updated the configuration files first, please use an older configuration files version that match the release date or change:

![image](https://user-images.githubusercontent.com/2745567/189683330-4010301d-2458-4c5e-912e-244fb2d58e19.png)

and

![image](https://user-images.githubusercontent.com/2745567/189683396-b1c73c17-e363-4ac9-9334-8addcb480cbc.png)

to `02010100`  <<-----  use this.

**Reply by MaynardMann** [reply]
awesome should I just be editing the version # in the lines and if so what version should It be changed to.
